cargo-process.el

```diff
@@ -742,6 +747,15 @@ Requires Cargo Script to be installed."
 					 " "
 					 buffer-file-name)))
 
+;;;###autoload
+(defun cargo-process-watch ()
+  "Run the Cargo watch command.
+With the prefix argument, modify the command's invocation.
+Cargo: Watches over your Cargo project’s source.
+Requires Cargo Watch to be installed."
```

Comment:
That's what I indented to do. But I'm following the style of the context, since you wrote almost all those "Cargo Audit", "Cargo Script" and a few more like this.

I pushed another commit the clean this style.